### PR TITLE
javascript-local-storage - Khanh Nguyen

### PR DESCRIPTION
**Pull request by khanhnguyenq**
https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/77e9192c-cf9c-4bd8-b646-7713e5e3550d

### CODE REVIEW

File: javascript-local-storage/data.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+let todos = [];
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== 'null') {
```
Comment (shawnkost) [inline javascript-local-storage/data.js]: No quotes around `null`. Now you're checking for a string of `'null'` rather than `null` itself

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Khanh

File: javascript-local-storage/data.js (hunk 1 above)
Comment (khanhnguyenq) [reply javascript-local-storage/data.js]: Thank you Shawn! Re-pushed to github and screenshot below.
Screenshot 2023-11-13 at 08 39 37

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**